translations/en.json

```diff
@@ -1354,6 +1354,7 @@
     "welcome-community-blank-message": "Your channels will appear here. To create a new channel, click on the ⊕ button and select \"Create a channel\"",
     "welcome-community-blank-message-edit-chats": "Your channels will appear here. To create a new channel, go back to the community screen, click on the ⊕ button and select \"Create a channel\"",
     "welcome-blank-community-message": "Your communities will appear here.",
+    "share-images-with-your-contacts": "Your contacts will appear here. To share an image with you contacts, add some contacts in your profile, share your images with your contacts.",
```

Comment:
where from did this copy come?

Reply:
I created this to show some message when the user sharing a message has empty contacts

Reply:
cc @hesterbruikman